utilities/install.py

```diff
@@ -0,0 +1,90 @@
+import os
+import subprocess
+import sys
+from shutil import copyfile
+
+branch = None
+
+
+# by default clones branch (which can be passed as a parameter python install.py branch test_branch)
+# if branch doesnt exist clones the default_branch
+
+def clone(repository, folder, default_branch, cwdp='', recursive=False, destination_folder=None):
+    global branch
+    print("Cloning " + repository)
+    if recursive:
+        subprocess.call(['git', 'clone', '--recursive', repository], cwd='./' + cwdp)
+    else:
+        if destination_folder:
+            subprocess.call(['git', 'clone', repository, destination_folder], cwd='./' + cwdp)
+        else:
+            subprocess.call(['git', 'clone', repository], cwd='./' + cwdp)
+    checkout(folder, default_branch, cwdp)
+
+
+def checkout(folder, default_branch, cwdp):
+    currentPath = os.getcwd()
+    print(currentPath)
+    newPath = currentPath + "/" + cwdp + folder
+    print(newPath)
+    os.chdir(newPath)
+    python_git = subprocess.Popen("git branch -a", shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    outstd, errstd = python_git.communicate()
+    if branch and branch in str(outstd):
+        subprocess.call(['git', 'checkout', branch], cwd='./')
+    else:
+        subprocess.call(['git', 'checkout', default_branch], cwd='./')
+    os.chdir(currentPath)
+
+
+def main(argv):
+    global branch
+    if (len(argv) > 0):
+        if (argv[0] == 'branch'):
+            branch = argv[1]
+
+
+if __name__ == "__main__":
+    main(sys.argv[1:])
+
+os.chdir(os.getcwd() + "/../")
+# Cloning Repos
+clone('https://github.com/openworm/pygeppetto.git', 'pygeppetto', 'development')
+subprocess.call(['pip', 'install', '-e', '.'], cwd='./pygeppetto/')
+
+clone('https://github.com/Neurosim-lab/netpyne.git', 'netpyne', 'ui')
+subprocess.call(['pip', 'install', '-e', '.'], cwd='./netpyne/')
+
+clone('https://github.com/openworm/org.geppetto.frontend.jupyter.git', 'org.geppetto.frontend.jupyter', 'refactor-sync')
+subprocess.call(['npm', 'install'], cwd='./org.geppetto.frontend.jupyter/js')
+subprocess.call(['npm', 'run', 'build-dev'], cwd='./org.geppetto.frontend.jupyter/js')
+
+# subprocess.call(['git', 'submodule', 'update', '--init'], cwd='./')
+clone('https://github.com/openworm/org.geppetto.frontend.git', 'geppetto', 'refactor-sync', 'hnn_ui/', False, 'geppetto')
+# checkout('geppetto', 'development','org.geppetto.frontend.jupyter/src/jupyter_geppetto/')
+clone('https://github.com/MetaCell/geppetto-hnn.git', 'geppetto-hnn', 'skeleton',
```

Comment:
@afonsobspinto Change skeleton to development after https://github.com/MetaCell/geppetto-hnn/pull/54
```suggestion
clone('https://github.com/MetaCell/geppetto-hnn.git', 'geppetto-hnn', 'development',
```

Reply:
👍